Settings.gd

```diff
@@ -54,10 +54,11 @@ var start_with_open_menu: bool = false
 
 func _ready():
 	var file = File.new()
-	file.open(CONFIG_PATH, file.READ)
-	var text = file.get_as_text()
-	var datum = JSON.parse(text)
-	self._config_data = JSON.parse(text).result
+	# If the file is not present or could not be opened, default settings are used.
+	if file.open(CONFIG_PATH, file.READ) == OK:
+		var text = file.get_as_text()
+		var datum = JSON.parse(text)
+		self._config_data = JSON.parse(text).result
```

Comment:
You are calling JSON.parse() twice here, on the line above you are setting datum but then never using it.

Reply:
Good catch. Not sure why that is in there. I'll remove that line.